examples/large_files/dfx.json

```diff
@@ -0,0 +1,29 @@
+{
+    "canisters": {
+        "large_files": {
+            "type": "custom",
+            "main": "src/index.ts",
+            "candid": "src/index.did",
+            "candid_gen": "http",
+            "build": "npx azle large_files",
+            "post_install": "npx azle upload-assets large_files",
+            "wasm": ".azle/large_files/large_files.wasm",
+            "gzip": true,
+            "assets_large": [
```

Comment:
Make an issue discussing if we should combine assets and assets_large

Reply:
https://github.com/demergent-labs/azle/issues/1702